Source/RunActivity/Viewer3D/Lights.cs

```diff
@@ -149,6 +154,15 @@ public LightViewer(Viewer viewer, TrainCar car, TrainCarViewer carViewer)
             UpdateActiveLightCone();
         }
 
+        string DefineFullTexturePath(string textureName)
+        {
+            if (File.Exists(Path.Combine(Path.GetDirectoryName(Car.WagFilePath), textureName)))
+                return Path.Combine(Path.GetDirectoryName(Car.WagFilePath), textureName);
+            if (File.Exists(Path.Combine(Viewer.ContentPath, textureName)))
+                return Path.Combine(Viewer.ContentPath, textureName);
+            return Path.Combine(Viewer.ContentPath, "LightGlow.png");
```

Comment:
It may help if a warning message is added to the log when we reach this line, as it means the file defined couldn't be found. On my first attempt, I formatted the texture path slightly incorrectly and had to look at a debugger to see what exactly was wrong. But if I had an error message like "Could not find light graphic XYZ at 'wagon file path\\light path you entered'" it would have been much faster to see my mistake.

Reply:
I'll add something similar.